website/docs/components/text/partials/code/how-to-use.md

```diff
@@ -0,0 +1,137 @@
+The `Text` component is a **code-only** component used to apply specific predefined HDS typographic styles to a block of text or content.
+
+The equivalent if Figma is not a component, but a set of [typographic Figma styles](https://www.figma.com/file/oQsMzMMnynfPWpMEt91OpH/HDS-Product---Foundations?type=design&node-id=1262-9192).
+
+## How to use this component
+
+The “base” `Text` component is specialized in three different variants–the ones actually used by the consumers–with a one-to-one association between the namespace (eg. `::Display`) and the [corresponding style in Figma](https://www.figma.com/file/oQsMzMMnynfPWpMEt91OpH/HDS-Product---Foundations?type=design&node-id=1262-9192&mode=design).
+
+```handlebars
+<Hds::Text::Display>Text as "Display" variant</Hds::Text::Display>
+<Hds::Text::Body>Text as "Body" variant</Hds::Text::Body>
+<Hds::Text::Code>Text as "Code" variant</Hds::Text::Code>
+```
+
+Since no `@tag` argument is provided (see below) by default the text element is rendered as a `<span>`.
+
+### HTML tag
+
+To specify which HTML tag to use to render the element, use the `@tag` argument:
+
+```handlebars
+<Hds::Text::Display @tag="h1">Text as "Display" variant</Hds::Text::Display>
+<Hds::Text::Body @tag="p">Text as "Body" variant</Hds::Text::Body>
+<Hds::Text::Code @tag="pre">Text as "Code" variant</Hds::Text::Code>
+```
+
+!!! Insight
+
+While by default the component renders as `<span>`, we invite consumers to:
+- consider which HTML semantic tag is the correct one for the context in which the text is used
+- always set a `@tag` value accordingly, to make the explicit the semantic choice
+
+!!!
+
+### Size
+
+To specify which size to apply to the text, use the `@size` argument:
+
+
+```handlebars
+<Hds::Text::Display @tag="h1" @size="500">Text as "Display-500" variant</Hds::Text::Display>
+<Hds::Text::Body @tag="p" @size="300">Text as "Body-300" variant</Hds::Text::Body>
+<Hds::Text::Code @tag="pre" @size="100">Text as "Code-100" variant</Hds::Text::Code>
+```
+
+Different style variants support different sizes: refer to the [Component API](#component-api) section for details.
```

Comment:
This seems duplicative of the critical banner in the next section

Reply:
They're not the same: one is about the sizes, one about the weight (they're two distinct arguments, in code, while in Figma the size is implicit with the style name (eg. `sisplay-500`) )